### PR TITLE
Topic/openms2.5 fix bool

### PR DESCRIPTION
**Pull request by bernt-matthias**
fixes https://github.com/galaxyproteomics/tools-galaxyp/issues/495

TODO

- [x] check discarded tool tests
- [x] remove 404 links from help (maybe with patch)

### CODE REVIEW

**Comment by bernt-matthias** [comment]
I will have a look why the order of the selects changes all the time .. this may be a bit annoying for users.

**Comment by bgruening** [comment]
Looks like there is a dict() involved?

**Comment by bernt-matthias** [comment]
@bgruening .. ready from my side. 

if you like we can change the number of chunks to 10 again for the PR workflow

**Comment by bgruening** [comment]
Is the select order now stable?

File: tools/openms/macros.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
      add additional macro files using the m/macros parameter -->
 <macros>
   <token name="@TOOL_VERSION@">2.5</token>
-  <token name="@GALAXY_VERSION@">0</token>
+  <token name="@GALAXY_VERSION@">1</token>
```
Comment (bgruening) [inline tools/openms/macros.xml:7]: do we want to increase it or patch the older version?

Comment (bernt-matthias) [reply tools/openms/macros.xml:7]: whatever you think is better.

**Comment by bernt-matthias** [comment]
> Is the select order now stable?

I did not find an obvious reason for the change of the order .. unfortunately I do not have time at the moment to work on this...

**Comment by bgruening** [comment]
Merci!

**Pull request merged**